hnn_core/network.py

```diff
@@ -1187,6 +1187,160 @@ def add_connection(self, src_gids, target_gids, loc, receptor,
 
         self.connectivity.append(deepcopy(conn))
 
+    def pick_connection(self, src_gids=None, target_gids=None,
+                        loc=None, receptor=None):
+        """Returns indices of connections that match search parameters.
+
+        Parameters
+        ----------
+        src_gids : str | int | range | list of int | None
+            Identifier for source cells. Passing str arguments
+            ('L2_pyramidal', 'L2_basket', 'L5_pyramidal', 'L5_basket') is
+            equivalent to passing a list of gids for the relvant cell type.
+            source - target connections are made in an all-to-all pattern.
+        target_gids : str | int | range | list of int | None
+            Identifer for targets of source cells. Passing str arguments
+            ('L2_pyramidal', 'L2_basket', 'L5_pyramidal', 'L5_basket') is
+            equivalent to passing a list of gids for the relvant cell type.
+            source - target connections are made in an all-to-all pattern.
+        loc : str | list of str | None
+            Location of synapse on target cell. Must be
+            'proximal', 'distal', or 'soma'. Note that inhibitory synapses
+            (receptor='gabaa' or 'gabab') of L2 pyramidal neurons are only
+            valid loc='soma'.
+        receptor : str | list of str | None
+            Synaptic receptor of connection. Must be one of:
+            'ampa', 'nmda', 'gabaa', or 'gabab'.
+
+        Returns
+        -------
+        conn_indices : list of int
+            List of indices corresponding to items in net.connectivity.
+            Connecion indices are added if any of the provided parameter
```

Comment:
```suggestion
            Connecion indices are included if any of the provided parameter
```

somehow on a quick read, "added" gave me the impression that `net.connectivity` had a new item